README.md

```diff
@@ -37,143 +37,160 @@ pre-commit install
+The batch sequence can be generated using [Pulser](https://github.com/pasqal-io/Pulser). See their [documentation](https://pulser.readthedocs.io/en/stable/),
+for more information on how to install the library and create your own sequence.
+
+The sequence should be a pulser Sequence object, once you have created it, you should serialize it as follows:
 
-You can create a workload through the SDK with the following command:
 ```python
-workload=sdk.create_workload(workload_type="<WORKLOAD_TYPE>",backend="<BACKEND>",config={"config_param_1":"value"})
+serialized_sequence = sequence.to_abstract_repr()
+```
 
-#You can cancel the workload by doing:
-sdk.cancel_workload(workload.id)
+Once you are login and have serialized your sequence, the last step is to define your jobs.
 
-#Or refresh the workload status/results by with the following:
-workload=sdk.get_workload(workload.id)
+When creating a job, select a number of runs and set the desired values for the variables defined in the sequence.
 
-#Once the workload has been processed you can fetch the result like this:
-print(workload.result)
+```python
+job1 = {"runs": 20, "variables": {"omega_max": 6}}
+job2 = {"runs": 50, "variables": {"omega_max": 10.5}}
 ```
 
+Then, send the batch of jobs to the QPU and wait for the results:
 
-## Advanced usage
+```python
+batch = sdk.create_batch(serialized_sequence, [job1,job2], wait=True)
+```
 
-### Authentication
+You can also choose to run your batch on an emulator using the optional argument `emulator`.
+For using a basic single-threaded QPU emulator that can go up to 10 qubits, you can specify the "EMU_FREE" emulator.
 
-There are several ways to provide a correct authentication using the SDK.
+```python
+from pasqal_cloud.device import EmulatorType
+batch = sdk.create_batch(
+    serialized_sequence, [job1, job2], emulator=EmulatorType.EMU_FREE
+)
+```
+
+Once the QPU has returned the results, you can access them with the following:
 
 ```python
-from pasqal_cloud import SDK
+for job in batch.ordered_jobs:
+    print(f"job-id: {job.id}, status: {job.status}, result: {job.result}")
+```
 
-project_id="your_project_id" # Replace this value by your project_id. It can be found on the user portal under the name "group_id". "Groups" have recently been renamed to "projects" and frontend is currently being updated accordingly.
+### Create a workload
 
-username="your_username" # Replace this value by your username or email on the PASQAL platform.
-password="your_password" # Replace this value by your password on the PASQAL platform.
-# Ideally, do not write this password in a script but provide in through the command-line or as a secret environment variable.
+A workload is a single unit of work to be computed by emulators or qpu.
+It is defined by a type (qadence_circuit, algorithm_dqgm, …) and a config.
+This config must be generated with the appropriate library for the desired type.
```

Comment:
Comment: This entire description reads quite vague, is this really the information we want to display?

Reply:
@MatthieuMoreau0 @awennersteen  Is it enought you think ?

Reply:
A rewording suggestion:

```
A workload is a unit of work to be executed on Pasqal Cloud Services infrastucture.  To submit a new workload, select a type of workloads, target one of the available backends and provide a configuration object to execute it.
``` 

The last sentence is vague maybe we could tell the user to refer to user portal documentation for details on workload configurations. But unfortunately that page doesn't exist just yet..

Reply:
I do agree it is a bit vague, should we list the current option and explain them very briefly ?